cypress/integration/accessibility_Icon_spec.js

```diff
@@ -5,6 +5,15 @@ describe('Icon Accessibility check', () => {
   });
 
   it('Tests accessibility on the Icon page', () => {
+    cy.configureAxe({
+      rules: [
+        {
+          id: 'aria-command-name',
```

Comment:
Can we fix the violations instead of disabling the rule? If not, can you add a comment here about the issue(s)?

Reply:
I'll document it better. The problem is that sometimes the cmp has an empty label because there's a wrapper giving it a name, such as a tooltip.  And I think this error come from there.